src/block-library/grid/row/generateRowStyles.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Import andt components, dependencies & configuration
+ */
+import { screenSizes } from "../../../_config";
+import { theme } from "antd";
+
+
+// Generates padding for child elements based on parent gutterValue
+export const generateRowStyles = (styles, clientId) => {
```

Comment:
@Danishkhurshid So it's clear, let's update the prop name to gutterArray, since that's actually what we're passing, not the whole styles object.